app/plugin.go

```diff
@@ -423,8 +409,115 @@ func (a *App) GetPlugins() (*model.PluginsResponse, *model.AppError) {
 	return resp, nil
 }
 
-// GetMarketplacePlugin returns plugin from marketplace-server
-func (a *App) GetMarketplacePlugin(request *model.InstallMarketplacePluginRequest) (*model.BaseMarketplacePlugin, *model.AppError) {
+func (a *App) InstallMarketplacePlugin(request *model.InstallMarketplacePluginRequest) (*model.Manifest, *model.AppError) {
```

Comment:
@ali-farooq0 Should we move this to `plugin_install.go` to keep all app installation code there?

Reply:
Good idea @marianunez. Makes sense to keep all the installation code in `plugin_install.go` 👍